CHANGELOG.md

```diff
@@ -2,11 +2,9 @@
 
 All notable changes to the `python-domino` library will be documented in this file.
 
-## [Unreleased]
```

Comment:
Can you do it like this? https://github.com/dominodatalab/python-domino/pull/120/files

So there's "Unreleased" section for next releases.